x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/delete/delete_action.tsx

```diff
@@ -14,9 +14,13 @@ import { ConfirmDeleteEndpointModal } from './confirm_delete_endpoint';
 
 interface DeleteActionProps {
   selectedEndpoint: InferenceEndpointUI;
+  preconfiguredEndpoint: boolean;
```

Comment:
I think we should just name this `disabled` or we can just do the `PRECONFIGURED_ENDPOINTS.includes(selectedEndpoint.endpoint)` in this component.

Reply:
I think this is better. I will update as suggested.